### ISSUE TITLE
Callouts are not showing with CustomCalloutView if there are two pins on same location

### ISSUE THREAD

**Issue by harishchopra86** [venits/react-native-map-clustering#126]
Callouts are not showing in latest version when there are two markers on same location.
When I tap on a pin with spiral view, nothing is happening.

Here is my code for adding markers:
```
{this.state.pins.map(pin => (
            <Marker
              coordinate={pin.location}
              key={pin.identifier}>
              <Callout>
                <CustomCalloutView
                  key={pin.identifier}
                  calloutTapAction={this.onCalloutButtonTapped}
                  row={pin} />
              </Callout>
            </Marker>
          ))}
```
It is working fine if there is one pin on a location.

**Comment by harishchopra86** [comment]
@chetanparmar95 Any idea/thoughts about this issue?

**Comment by harishchopra86** [comment]
@venits Any suggestion?

**Comment by harishchopra86** [comment]
@venits @chetanparmar95 
Any update/fix on this?

**Comment by nbasael** [comment]
Im having the exact same issue, my code looks like this:

```jsx
  <MapView
    provider='google'
    showsCompass={true} 
    initialRegion={INITIAL_REGION}
    style={styles.mapa}
    rotateEnabled={false}
    pitchEnabled={false}
    toolbarEnabled={false}
    loadingEnabled={true}
    mapRef={map => { this.map = map }}
    customMapStyle={EstiloMapaHome}
    clusterColor='#014AAAFF'
    onMapReady={() => {
      this.map.fitToCoordinates(this.props.arrayPosiciones, {
        edgePadding: { top: 150, right: 130, bottom: 150, left: 130 },
        animated: false
      });
    }}
  >
    {
      this.props.datos.map((v, i) => (
        <Marker
          coordinate={{
            latitude: v.lat,
            longitude: v.lon,
          }}
          pinColor={v.activo ? '#5cb85c' : '#d9534f'}>
          <Callout tooltip>
            <Card>
              <Card.Content>
                <Title>{v.nombre}</Title>
                <Divider />
                <View style={{ alignItems: 'center', flexDirection: 'row', paddingTop: 5 }}>
                  <Paragraph>Status:</Paragraph>
                  {
                    // Some conditional rendering
                  }
                </View>
              </Card.Content>
            </Card>
          </Callout>
        </Marker>
      ))
    }
  </MapView>
```

**Comment by vpwhitefox46** [comment]
Fixed in this PR: #146 
it's not yet merged.

**Comment by venits** [comment]
Fixed it version `3.2.0` 🎉

Thank you for your contribution ❤️
